spec/request/users_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  describe 'GET #index' do

```

Comment:
- [ ] Your spec files should include tests that ensure the response body includes the correct placeholder text. 
- Example : `expect(response.body).to include("Welcome to user index page")`, 
- Kindly, let's add this to the `index` and `show` actions for the `User` and `Post` controllers